### ISSUE TITLE
content preloaded on node{1-3}.preload.ipfs.io is not available from ipfs.io gateway

### ISSUE THREAD

**Issue by PierreJeanjacquot** [ipfs/js-ipfs#3577]
- **Version**:
```js
{
  version: '0.5.2',
  repo: 10,
  commit: 'ff76e7cda441d66d327e03e3ca1dea6e901ec918',
  'interface-ipfs-core': '^0.144.1'
}
```

- **Platform**:
Linux Ubuntu 18.04
Nodejs v14.4.0 

- **Subsystem**:
ipfs-core/prelaod

#### Severity:
Medium

#### Description:

Adding a file to IPFS using `ipfs.add()` with **preload enabled**.
Depending on the preload node used the file can or can NOT be fetched from the `ipfs.io` gateway.
Expected preload to work with all preload nodes.
This is probably not an error in JS side but reporting here as the lib uses these preload nodes as default setup. 

#### Steps to reproduce the error:

Use this script to add random content to IPFS and preload it on a specific node.
```js
const { randomBytes } = require("crypto");
const Ipfs = require("ipfs");
const Debug = require("debug");
const fetch = require("node-fetch");

Debug.enable("ipfs:preload*");

const preloadRandomBytes = async preloadMultiaddr => {
  console.log(`stating ipfs with preload on ${preloadMultiaddr}`);
  const ipfs = await Ipfs.create({
    preload: {
      enabled: true,
      addresses: [preloadMultiaddr]
    }
  });

  const buffer = Buffer.from(randomBytes(8));

  const result = await ipfs.add(buffer);
  const { cid } = result;
  console.log("file added to ipfs", cid.toString());

  console.log(`fetching ${cid.toString()} from ipfs.io gateway`);
  await fetch(`https://ipfs.io/ipfs/${cid.toString()}`).then(res => {
    if (!res.ok) throw Error("ipfs.io bad response");
    console.log(`found ${cid.toString()} on ipfs.io gateway`);
  });

  await ipfs.stop();
};

const preloadMultiaddr = process.argv[2];

preloadRandomBytes(preloadMultiaddr);
```

run `node script.js "/dns4/node0.preload.ipfs.io/https"` => content is preloaded to `node0.preload.ipfs` and fetched from `ipfs.io` :heavy_check_mark: 
run `node script.js "/dns4/node1.preload.ipfs.io/https"` => content is preloaded to `node1.preload.ipfs` but fetch from `ipfs.io` hangs forever :x: 
same issue with `node2.preload.ipfs` and `node3.preload.ipfs` :x:

**Comment by aschmahmann** [comment]
Investigation steps:
- Is this issue still on going?
- Are all preload nodes peered to gateways (infra question)?
- Are the gateways having trouble fetching data from the preload nodes? (e.g. go-ipfs issue)

**Comment by Stebalien** [comment]
We've restarted the preload nodes which may have (temporarily) fixed this. Otherwise, this is likely a bug in go-bitswap.

**Comment by BigLep** [comment]
@Stebalien: Apologies if misunderstanding, but it sounded like from verbal conversations that there may be a known go-bitswap issue that could be at play.  Can we please link to it?

**Comment by PierreJeanjacquot** [comment]
@Stebalien 
> We've restarted the preload nodes which may have (temporarily) fixed this. Otherwise, this is likely a bug in go-bitswap.

All 4 preload nodes are currently working as expected.

**Comment by Stebalien** [comment]
> @Stebalien: Apologies if misunderstanding, but it sounded like from verbal conversations that there may be a known go-bitswap issue that could be at play. Can we please link to it?

I looked through the old issues, and none of them looked like they might have caused this one. I've filed a new issue (https://github.com/ipfs/go-ipfs/issues/7972) but it may be a while before we can actually debug and figure it out.

(closing as the preload nodes are now working and this isn't an issue with js-ipfs)

**Comment by PierreJeanjacquot** [comment]
Hi @Stebalien I went into the same issue again today, `node2.preload.ipfs.io` is not working as expected.
